.github/workflows/inductor-perf-test-nightly.yml

```diff
@@ -0,0 +1,36 @@
+name: inductor
+
+on:
+  schedule:
```

Comment:
How do we test this PR? I think adding a trigger with the `ciflow/nightly` label would be helpful here.

Reply:
Yes, I will add the ciflow/nightly trigger. Also I just realized the name conflict "inductor", I should rename it to inductor-perf-test.